pysrc/bytewax/connectors/kafka.py

```diff
@@ -42,29 +42,50 @@ def _list_parts(client, topics):
 
 
 class _KafkaSource(StatefulSource):
-    def __init__(self, consumer, topic, part_idx, starting_offset, resume_state):
+    def __init__(
+        self,
+        consumer,
+        topic,
+        part_idx,
+        starting_offset,
+        resume_state,
+        batch_size,
+        timeout,
```

Comment:
So, I added this parameter because during tests, making the timeout longer (0.1 seconds instead of 0.001) did help with batching when using a really high messages per second count. But I was purely testing kafkainput to nulloutput, so I didn't see any kind of degradation to the rest of the worker, which will happen as soon as there is other work to do. I'll remove the parameter.
I'm really liking the idea of the `next_awake` method in input sources, but I agree it doesn't belong to this PR.